src/views/projects/Issue.svelte

```diff
@@ -119,58 +204,102 @@
   <header>
     <div class="summary">
       <div class="summary-title txt-medium">
-        {issue.title}
-      </div>
-      <div class="txt-monospace id layout-desktop">{issue.id}</div>
-      <div class="txt-monospace id layout-mobile">
-        {formatObjectId(issue.id)}
+        {#if titleEditable}
+          <TextInput transparent variant="form" bind:value={newTitle} />
+        {:else}
+          <span class="txt-medium">{issue.title}</span>
+          <div class="txt-monospace id layout-desktop">{issue.id}</div>
+          <div class="txt-monospace id layout-mobile">
+            {formatObjectId(issue.id)}
+          </div>
+        {/if}
       </div>
+      <Button
+        variant="text"
+        size="small"
+        on:click={async () => {
+          if (newTitle !== issue.title) {
+            await editTitle(newTitle);
+          }
+          titleEditable = !titleEditable;
+        }}>
+        {#if titleEditable}
+          save
+        {:else}
+          edit
+        {/if}
+      </Button>
+    </div>
+    <div class="summary-state">
       <div
-        class="summary-state"
+        class="label"
         class:closed={issue.state.status === "closed"}
         class:open={issue.state.status === "open"}>
-        {capitalize(issue.state.status)}
+        {#if issue.state.status === "open"}
+          {capitalize(issue.state.status)}
+        {:else}
+          {capitalize(issue.state.status)}
+          <span class="txt-tiny">
+            {issue.state.reason}
+          </span>
+        {/if}
       </div>
+      <Authorship
+        timestamp={issue.timestamp}
+        author={issue.author}
+        caption="opened this issue" />
     </div>
-    <Authorship
-      author={issue.author}
-      timestamp={issue.timestamp}
-      caption="opened" />
   </header>
-  <main>
+  <main class="form">
     <div class="comments">
       {#each issue.discussion as comment}
-        <Comment {comment} {getImage} />
+        {#if comment.replyTo === null}
+          <Reply
+            bind:reply
+            allowReply={Boolean(session)}
+            on:reply={() => createReply(comment.id, reply)}
```

Comment:
Do we need to both pass the reply via bind _and_ the event? If not I'd just use the event to pass it back.

Reply:
Yeah good catch, I think we can just keep the event.